casper/contracts/simple_casper.v.py

```diff
@@ -316,6 +321,18 @@ def total_prevdyn_deposits_in_wei() -> wei_value:
     return floor(self.total_prevdyn_deposits * self.deposit_scale_factor[self.current_epoch])
 
 
+
+@public
+# cannot be labeled @constant because of external call
+def validate_vote_msg(vote_msg: bytes[1024]) -> bool:
```

Comment:
Maybe we should call this `validate_vote_msg_signature` or just `validate_vote_signature`. To make sure to imply it is _not_ doing the `votable` validation.